openverse_catalog/dags/providers/provider_api_scripts/cleveland_museum_of_art.py

```diff
@@ -78,64 +79,59 @@ def _get_response(query_param, endpoint=ENDPOINT, retries=RETRIES):
         return response_json, total_images
 
 
+def get_int_value(data, key):
+    value = data.get(key)
+    if value is not None and value != "":
+        return int(value)
```

Comment:
Do we want to add a `try`/`except` here in case the value doesn't happen to be an `int`, and maybe return `None` in those cases? Or do you think it's worth halting execution of the provider script for that case?

Reply:
That's perfect!